libs/ui/src/lib/modules-graph/modules-graph.component.ts

```diff
@@ -14,14 +14,17 @@ import { FormatFileSizePipe } from '../format-file-size.pipe';
 
 interface Chunk {
   id: string;
-  name: string;
+  name: void | string;
```

Comment:
void?

Reply:
`undefined`, i mixed it up with Flowtype